docs/tasks.md

```diff
@@ -142,49 +119,68 @@ spec:
       emptyDir: {}
 ```
 
-### Steps
+### `Task` vs. `ClusterTask`
 
-The `steps` field is required. You define one or more `steps` fields to define
-the body of a `Task`.
+A `ClusterTask` is a `Task` scoped to the entire cluster instead of a single namespace.
+A `ClusterTask` behaves identically to a `Task` and therefore everything in this document
+applies to both.
 
-If multiple `steps` are defined, they will be executed in the same order as they
-are defined, if the `Task` is invoked by a [`TaskRun`](taskruns.md).
-Each `steps` in a `Task` must specify a container image that adheres to the
-[container contract](./container-contract.md). For each of the `steps` fields,
-or container images that you define:
+**Note:** When using a `ClusterTask`, you must add a `TaskRef`. 
```

Comment:
The original was very confusing, it said: "In case of using a ClusterTask, the TaskRef kind should be added. The default kind is Task which represents a namespaced Task", so it's very possible I misunderstood it. Can you untangle this for me?

Reply:
Actually, I think I figured it out! How about:

"**Note:** When using a `ClusterTask`, you must explicitly set the `kind` sub-field in the `taskRef` field to `ClusterTask`. If you don't specify this, the `kind` sub-field defaults to `Task.`

Reply:
Ahhhh I understand now.  Yeah you nailed it!

Reply:
Yay, fixed!